core/src/main/java/com/graphhopper/reader/dem/MultiSourceElevationProvider.java

```diff
@@ -93,10 +101,25 @@ public void release() {
         globalProvider.release();
     }
 
-    @Override
-    public void setAutoRemoveTemporaryFiles(boolean autoRemoveTemporary) {
+    /**
+     * Creating temporary files can take a long time as we need to unpack them as well as to fill
+     * our DataAccess object, so this option can be used to disable the default clear mechanism via
+     * specifying 'false'.
+     */
+    public MultiSourceElevationProvider setAutoRemoveTemporaryFiles(boolean autoRemoveTemporary) {
         srtmProvider.setAutoRemoveTemporaryFiles(autoRemoveTemporary);
         globalProvider.setAutoRemoveTemporaryFiles(autoRemoveTemporary);
+        return this;
+    }
+
+    @Override
+    String getFileName(double lat, double lon) {
+        return null;
+    }
+
+    @Override
+    String getDownloadURL(double lat, double lon) {
+        return null;
```

Comment:
If these two methods are not supposed to be called, i'd rather throw an `UnsupportedOperationException`

Reply:
OK @otbutz, done